### PR TITLE
lab1

### PR DESCRIPTION
**Pull request by KirylAkulich1**
ConsoleApplication

### CODE REVIEW

File: lab1/ChatApplication/ChatApplication/Message/Message.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+namespace ChatApplication.Messages
+{
+    public class Message
+    {
+        private string _contentHash;
```
Comment (gmltA) [inline lab1/ChatApplication/ChatApplication/Message/Message.cs:5]: Why do you need to store hash explicitly. And why do you need a setter for it?

File: lab1/ChatApplication/ChatApplication/Container.cs [hunk 2]
```diff
@@ -0,0 +1,53 @@
+using System.Runtime.CompilerServices;
+using ChatApplication.ConnectionStratagies;
+using ChatApplication.ConnectivityService;
+using ChatApplication.Loggers;
+using ChatApplication.MessageService;
+
+namespace ChatApplication
+{
+    //Simple IoC container).Ne beite
```
Comment (gmltA) [inline lab1/ChatApplication/ChatApplication/Container.cs:9]: Extremely simple I'd say.

File: lab1/ChatApplication/ChatApplication/Message/Message.cs (hunk 1 above)
Comment (KirylAkulich1) [reply lab1/ChatApplication/ChatApplication/Message/Message.cs:5]: To check if message content was recieved correctly

Comment (gmltA) [reply lab1/ChatApplication/ChatApplication/Message/Message.cs:5]: Okay, but why don't you access stored private value in a getter of `ContentHash`? You literally re-calculate it each time instead. Or am I missing something?

Comment (KirylAkulich1) [reply lab1/ChatApplication/ChatApplication/Message/Message.cs]: Oooops....I understand,it's mistake.

**Review by gmltA — APPROVED** [review]

**Pull request merged**